### PR TITLE
chore: Update Gatsby quickstart guide

### PR DESCRIPTION
**Pull request by LekoArts**
Hi!

Here are some improvements to the Gatsby quickstart guide, most notably correcting that its config file is `gatsby-config.ts` and not `gatsby.config.ts` :)

### CODE REVIEW

**Review by jescalan — APPROVED** [review]
🤩

File: src/pages/quickstarts/full-stack/gatsby.mdx [hunk 1]
```diff
@@ -72,7 +70,7 @@ export default config
 
 Clerk offers Control Components that allow you to protect your pages, below is a simple example using Clerk's hosted components and the `<SignedIn/>` and `<SignedOut/>` control components.
```
Comment (alexisintech) [inline src/pages/quickstarts/full-stack/gatsby.mdx:71]:
```suggestion
Clerk offers Control Components that allow you to protect your pages. Below is a simple example using Clerk's hosted components and the `<SignedIn />` and `<SignedOut />` control components.
```

File: src/pages/quickstarts/full-stack/gatsby.mdx [hunk 2]
```diff
@@ -33,18 +33,18 @@ Once you have a React application ready, you need to install Clerk's React SDK.
 
 ### Set Environment Keys
 
-Below is an example of your .env.development file. To get the respective keys go to the API Keys page in the Clerk dashboard.
+Below is an example of your `.env.development` file. To get the respective keys go to the API Keys page in the Clerk dashboard.
```
Comment (alexisintech) [inline src/pages/quickstarts/full-stack/gatsby.mdx:36]:
```suggestion
Below is an example of your `.env.development` file. To get the respective keys, go to the API Keys page in the Clerk dashboard.
```

File: src/pages/quickstarts/full-stack/gatsby.mdx [hunk 3]
```diff
@@ -33,18 +33,18 @@ Once you have a React application ready, you need to install Clerk's React SDK.
...
 ```sh copy filename=".env.development"
 GATSBY_CLERK_PUBLISHABLE_KEY=pk_test_••••••••••••••••••••••••••••••••••
 CLERK_SECRET_KEY=sk_test_••••••••••••••••••••••••••••••••••
 ```
 
-### Update gatsby.config.ts
+### Update gatsby-config.ts
 
-To use Clerk you will need to update your `gatsby.config.ts` to resolve the `gatsby-plugin-clerk` and load from `.env` files.
+To use Clerk you will need to update your `gatsby-config.ts` to resolve `gatsby-plugin-clerk` and load from `.env` files.
```
Comment (alexisintech) [inline src/pages/quickstarts/full-stack/gatsby.mdx:45]:
```suggestion
To use Clerk, you will need to update your `gatsby-config.ts` in order to resolve `gatsby-plugin-clerk` and load from `.env` files.
```

**Review by alexisintech — CHANGES_REQUESTED** [review]
Just some grammar/editorial adjustments to make sure the content is polished 😄

**Review by alexisintech — APPROVED** [review]
😸💖

**Comment by BRKalow** [comment]
Updated and merged in https://github.com/clerkinc/clerk-docs/pull/167, thank you for your contribution!

**Pull request closed without merging**